dali/python/backend_impl.cc

```diff
@@ -1551,7 +1551,7 @@ PYBIND11_MODULE(backend_impl, m) {
           py::detail::make_caster<Tensor<CPUBackend>&> conv;
           bool is_cpu_data = conv.load(static_cast<py::object>(list[0]), true);
           if (is_cpu_data) {
-            FeedPipeline<CPUBackend>(p, name, list, 0, true);
+            FeedPipeline<CPUBackend>(p, name, list, AccessOrder::host(), true);
```

Comment:
Don't you have to adjust `.def("SetExternalTLInput",` as well? We get the the exact stream there as an agument of `SetExternalInput`.

Reply:
Will check.

Reply:
It doens't seem so - there's no UseStream stuff there (though perhaps there should be...).

Reply:
It doesn't seem so - the distinction between CPU and GPU input is done by type there, so there's no is_cpu_data.

Reply:
I was asking if the `stream` that is passed i L1530 is converted to `AccessOrder` automatically or you need to create `AccessOrder` there from it?

Reply:
It is converted automatically.